app/models/btc_payment.rb

```diff
@@ -15,4 +15,18 @@ def self.payments_before_date(date)
     BtcPayment.where('created_at < ?', date)
   end
 
+  def self.create_entry(destination, public_address, float_amount)
```

Comment:
Unused method argument - `float_amount`. If it's necessary, use `_` or `_float_amount` as an argument name to indicate that it won't be used.